Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerSuggestionsCommandsHeaderView.swift

```diff
@@ -5,26 +5,36 @@
 import StreamChat
 import UIKit
 
+/// The header reusable view of the suggestion collection view.
 public typealias ChatMessageComposerSuggestionsCommandsReusableView =
     _ChatMessageComposerSuggestionsCommandsReusableView<NoExtraData>
 
+/// The header reusable view of the suggestion collection view.
 open class _ChatMessageComposerSuggestionsCommandsReusableView<ExtraData: ExtraDataTypes>: UICollectionReusableView,
     UIConfigProvider {
+    
+    /// The reuse identifier of the reusable header view.
     class var reuseId: String { String(describing: self) }
-
+    
+    /// The suggestions header view.
     public lazy var suggestionsHeader: _ChatMessageComposerSuggestionsCommandsHeaderView<ExtraData> = {
         let header = uiConfig.messageComposer.suggestionsHeaderView.init().withoutAutoresizingMaskConstraints
         embed(header)
         return header
     }()
 }
 
+/// The header view of the suggestion collection view.
 public typealias ChatMessageComposerSuggestionsCommandsHeaderView = _ChatMessageComposerSuggestionsCommandsHeaderView<NoExtraData>
 
+/// The header view of the suggestion collection view.
 open class _ChatMessageComposerSuggestionsCommandsHeaderView<ExtraData: ExtraDataTypes>: _View, UIConfigProvider {
+    
+    /// The image icon of the commands header view.
     public private(set) lazy var commandImageView = UIImageView()
```

Comment:
Explicit type?

Reply:
Do we really need the explicit type here? 🤔 It is already described right after

Reply:
Yeah, it might seem redundant, but let's make a habit out of it. We can even have a SwiftLint rule for that :) If you want to remove the duplicate naming you can do ` public private(set) lazy var commandImageView: UIImageView = .init()`